Xamarin.Forms.Core.UnitTests/CarouselViewTests.cs

```diff
@@ -73,25 +73,25 @@ public void TestPositionChangedEvent()
 			Assert.IsTrue(countFired == 1);
 		}
 
-		[Test]
-		public void TestCurrentItemChangesPosition()
-		{
-			var gotoPosition = 1;
-			var source = new List<string> { "1", "2", "3" };
-			var carouselView = new CarouselView
-			{
-				ItemsSource = source
-			};
+		//[Test]
```

Comment:
We now set the CurrentItemAndPosition from the renderers

Reply:
If we don't need the test, remove it. If we do, uncomment it.

Reply:
Fixed